rig.js

```diff
@@ -60,26 +60,30 @@ class RigManager {
   }
 
   async setLocalAvatarUrl(url) {
-    await this.localRigQueue.lock();
-
-    const o = await new Promise((accept, reject) => {
-      new GLTFLoader().load(url, accept, xhr => {}, reject);
-    });
-    o.scene.traverse(o => {
-      if (o.isMesh) {
-        o.frustumCulled = false;
-      }
-    });
-    this.scene.remove(this.localRig.model);
-    this.localRig = new Avatar(o, {
-      fingers: true,
-      hair: true,
-      visemes: true,
-      // decapitate: selectedTool === 'firstperson',
-    });
-    this.scene.add(this.localRig.model);
-
-    await this.localRigQueue.unlock();
+    try {
```

Comment:
Why try?

Reply:
URL can bounce and was crashing site, uncaught promise rejection

Reply:
Then we should scope this down to the actual problematic fetch, rather than all errors we might see randomly.